styles/style.scss

```diff
@@ -0,0 +1,166 @@
+
+*{
+    box-sizing: border-box ;
+}
+.container{
+  position: relative;
+ 
+}
+.container_img{
+    width: 100%;
+    height: 550px;
+}
+.container_playbtn{
+    position: absolute;
+   margin-top: 150px;
+   right: 600px;
+}
+h1{
+    font-family: "Pacifico";
+    font-size: 60px;
+    position: absolute;
+    margin-top: -300px;
+    right: 500px;
+    color: aliceblue;
+    line-height: 1.2 ;
+}
+.second_button{
+    position: relative;
+    display: flex; 
```

Comment:
@sandesh9988 If you've declared flex, declare the flex-wrap too. This helps in cross-browser support.